src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
```

Comment:
i'm approving this, but please,
avoid nested ternary (i have some simular issue with my code right now)